src/js/videojs.record.js

```diff
@@ -1,5 +1,5 @@
 /**
- * @file videojs.record.js
+ * @FILE videojs.record.js
```

Comment:
i prefer lowercase.

Reply:
Didn't mean to push/commit, nice catch

fixed in rebase 2ea7122